### DISCUSSION TITLE
Data source has been added, but it needs some more configuration....

### DISCUSSION THREAD

**Discussion by CuriousDataLearner** [grafana/grafana-json-datasource#325] [Q&A]
I added the JSON API data source but when trying to save/test I get a (JSON API: Not Found) message.
I've tried adding my user and key into the query string like below:

- (curl -H "X-Auth-User: jdoe@church.com" -H "X-Auth-Key: a86850deb2742ec3cb41518e26aa2d89" https://churchmetrics.com/api/v1/users.json)

I've also tried adding them as custom HTTP Headers with X-Auth-User / X-Auth-Key as separate headers and values. 

I'm new, I don't understand how to configure the plug in data source. 
![image](https://user-images.githubusercontent.com/122706285/212551651-1a70e77c-f0a0-4df3-ad98-456cab0927d8.png)

**Comment by yesoreyeram** [comment]
How are you installing the plugin? seems installation issues.
